src/map.cpp

```diff
@@ -216,10 +216,11 @@ void Map::setNode(v3s16 p, MapNode & n)
 	// Never allow placing CONTENT_IGNORE, it fucks up stuff
 	if(n.getContent() == CONTENT_IGNORE){
 		bool temp_bool;
-		errorstream<<"Map::setNode(): Not allowing to place CONTENT_IGNORE"
-				<<" while trying to replace \""
-				<<m_nodedef->get(block->getNodeNoCheck(relpos, &temp_bool)).name
-				<<"\" at "<<PP(p)<<" (block "<<PP(blockpos)<<")"<<std::endl;
+		errorstream << "Map::setNode(): Not allowing to place CONTENT_IGNORE"
+			<< " while trying to replace \""
+			<< m_nodedef->get(block->getNodeNoCheck(relpos, &temp_bool)).name
+			<< "\" at " << PP(p) << " (block " << PP(blockpos) << ") "
+			<< "with an unrecognized node."<< std::endl;
```

Comment:
> […] an unrecognized node.

This is actually not correct, the node is indeed ignore at this place, not an unknown one.

Reply:
I guess usually it is ignore due to a nodename that isn't found as a registered node.
Problem is that ignore is being passed to 'setNode' so there is no easy way to get the nodename from here, you would have to pass more information to 'setNode' and i don't think that's worth doing.